cockroachcloud/serverless-faqs.md

```diff
@@ -106,13 +106,13 @@ When a {{ site.data.products.serverless }} cluster is idle, it will scale down t
 
 ### Is my cluster secure?
 
-Yes, we use separate certificate authorities for each cluster, and all connections to the cluster over the internet use TLS 1.3.
+Yes, we authenticate each cluster internally using certificate authorities, and all connections to the cluster over the internet use TLS 1.3.
 
 ### What certificates do I need to connect to my cluster?
 
-All connections to {{ site.data.products.serverless }} require SSL encryption. When connecting to your cluster using the CockroachDB SQL client or many drivers and ORMs, you don't need to download a root certificate and configure your client to use that certificate because the client will connect using the system root certificates. If you configure your client to use SSL and to verify the certificates (for example, by setting `sslmode=verify-full` in your [connection string](../{{site.current_cloud_version}}/connection-parameters.html#additional-connection-parameters)), your connection will be encrypted.
+All connections to {{ site.data.products.serverless }} require SSL encryption. When connecting to your cluster using the CockroachDB SQL client or many drivers and ORMs, you don't need to download a root certificate and configure your client to use that certificate because the client will connect using the system root certificates.
```

Comment:
```suggestion
All connections to {{ site.data.products.serverless }} require SSL encryption. When connecting to your cluster using the CockroachDB SQL client or many drivers and ORMs, you don't need to download a root CA certificate and configure your client to use that certificate because the client will connect using the system root CA certificates.
```